pkg/master/master.go

```diff
@@ -698,12 +704,14 @@ func (m *Master) getExtensionResources(c *Config) map[string]rest.Storage {
 		}
 		go func() {
 			wait.Forever(func() {
```

Comment:
optional nit, I know you didn't add this just now: you could just `go wait.Forever...`, there's no need for the wrapping function.